src/me/StevenLawson/TotalFreedomMod/Commands/Command_kick

```diff
@@ -0,0 +1,53 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import me.StevenLawson.TotalFreedomMod.Commands.Command_smite;
+import me.StevenLawson.TotalFreedomMod.TFM_AdminList;
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
+@CommandParameters(description = "Kick someone from the server", usage = "/<command> <playername> <reason>")
+public class Command_kick
+{
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        String reason = null;
+        if (args.length >= 2)
+        {
+            reason = StringUtils.join(ArrayUtils.subarray(args, 1, args.length), " ");
+        }
+            {
+                if (!TFM_AdminList.isSuperAdmin(sender))
```

Comment:
Should really check if the person that is trying to be kicked it of an equal or higher rank, if so then deny the ability. If they are a lower rank then allow them to kick.